backend/app/moderation.py

```diff
@@ -459,17 +500,28 @@ def submit_review(
             request.job_id, "Failed", "The review was rejected by a moderator."
         )
 
+    inform_only_moderators = False
+    verification_status = get_verification_status(request.appid)
+
+    if request.is_approved:
+        category = EmailCategory.MODERATION_APPROVED
+        subject = f"Change in build #{request.build_id} approved"
+    else:
+        category = EmailCategory.MODERATION_REJECTED
+        subject = f"Change in build #{request.build_id} rejected"
+
+    if not verification_status.verified:
```

Comment:
I'm not sure why this is the deciding factor here, as an app can still have developers with emails assigned, even if it's unverified.

Reply:
Marking as resolved as per what I sent on Matrix:

>people who submitted an app using github are more likely to never log in to the website
>so we don't have their e-mails, and they won't know a build was rejected
>while for direct upload, you have to log in to verify the ID before uploading anything